smoketests/tests/permissions.py

```diff
@@ -57,10 +49,9 @@ def test_logs(self):
     def test_publish(self):
         """This test checks to make sure that you cannot publish to an identity that you do not own."""
 
-        self.new_identity()
         self.publish_module()
 
-        self.reset_config()
+        self.new_identity()
```

Comment:
the goal of this `reset_config` was to switch identities, which we do directly now